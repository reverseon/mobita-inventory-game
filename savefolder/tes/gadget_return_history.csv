id;id_peminjaman;tanggal_pengembalian
12;1;23/10/2020
153;2;23/10/2021
185;3;23/10/1998
