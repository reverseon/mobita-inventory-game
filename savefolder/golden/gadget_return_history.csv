id;id_peminjaman;tanggal_pengembalian
9522;76853;21/12/2002
5002;76853;12/12/2004
120384;108662;20/10/2005
